mc-server-wrapper-lib/src/parse.rs

```diff
@@ -64,15 +64,22 @@ impl ConsoleMsgSpecific {
             ConsoleMsgSpecific::PlayerAuth { name, uuid }
         } else if console_msg.msg_type == ConsoleMsgType::Info
             && (console_msg.thread_name.starts_with("Async Chat Thread")
-                || console_msg.msg.starts_with('<') && console_msg.thread_name == "Server thread")
+                || console_msg.msg.starts_with('<')
+                || console_msg.msg.starts_with("[Not Secure] <")
+                    && console_msg.thread_name == "Server thread")
```

Comment:
We'll need to add some parentheses here:

```rust
|| (console_msg.msg.starts_with('<') || console_msg.msg.starts_with("[Not Secure] <"))
       && console_msg.thread_name == "Server thread")
```

Reply:
really? i always get fumbled up in boolean operation order.

Reply:
Yeah. Without parentheses, this code does:

```
a || (b && c)
```

but what we want is:

```
(a || b) && c
```

Reply:
got it. (do you know where i can read about this?)

Reply:
Here's one thread on the topic: https://www.reddit.com/r/rust/comments/elyu2h/what_is_the_precedence_of_boolean_operators/

MDN seems to have a page on it as well: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence